train/vpc/labs.py

```diff
@@ -137,6 +137,71 @@ def get_user_instance_info(conn, user_vpc, lab_tag, user):
     return final
 
 
+def ssh_config_instance(instance, user):
+    if not 'Name' in instance.tags.keys():
+        return ""
+    name = instance.tags['Name']
+    node = "-".join(name.split("-")[1:])
+    return """
+Host {0}
+     HostName {1}
+     User ubuntu
+     IdentityFile /host/{2}/users/{3}/{3}-{2}.pem""".format(node,
+                                                            instance.ip_address,
+                                                            VPC,
+                                                            user)                                                            
+
+def ssh_config(conn, user_vpc, user):
+    """Generate ssh_config for user"""
+    reservations = conn.get_all_instances(filters = {'vpc-id': user_vpc.id,
+                                                     'tag:User': user})
+    for r in reservations:
+        for instance in r.instances:
+            print ssh_config_instance(instance, user)
+
+def instance_to_inventory(instance):
+    if 'User' in instance.tags.keys():
+        return "{0} tagged=yes private_ip={1} dns_name={2} \
+ansible_ssh_private_key_file=/host/{3}/users/{4}/{4}-{3}.pem\n".format(instance.ip_address,
+                                           instance.private_ip_address,
```

Comment:
train/vpc/labs.py:166:44: E128 continuation line under-indented for visual indent
train/vpc/labs.py:167:44: E128 continuation line under-indented for visual indent
train/vpc/labs.py:168:44: E128 continuation line under-indented for visual indent
train/vpc/labs.py:169:44: E128 continuation line under-indented for visual indent